src/components/Notifications/utils.tsx

```diff
@@ -100,7 +100,7 @@ const NotificationComponentByType: DecentralandNotificationComponentByType<DCLNo
       CampaignOutOfStockNotification,
   }
 
-const CURRENT_AVAILABLE_NOTIFICATIONS = Object.values(NotificationType)
+const CURRENT_AVAILABLE_NOTIFICATIONS = Object.keys(NotificationComponentByType)
```

Comment:
is this change to validate in another app something like this:
```
if (CURRENT_AVAILABLE_NOTIFICATIONS.includes(A_NOTIFICATION)) {
   ....
}
```
???

Reply:
Exactly, it's being used here https://github.com/decentraland/decentraland-dapps/blob/feb4d6ef939d29be1506827f6cd09624630f6c4c/src/hooks/useNotifications.tsx#L91

Reply:
https://github.com/decentraland/decentraland-dapps/blob/feb4d6ef939d29be1506827f6cd09624630f6c4c/src/hooks/useNotifications.tsx#L91